src/renderer/webgl/WebglRenderer.ts

```diff
@@ -95,22 +193,59 @@ export class WebglRenderer extends EventEmitter implements IRenderer {
     // Detect whether IntersectionObserver is detected and enable renderer pause
     // and resume based on terminal visibility if so
     if ('IntersectionObserver' in window) {
-      const observer = new IntersectionObserver(e => this.onIntersectionChange(e[0]), { threshold: 0 });
+      const observer = new IntersectionObserver(e => this._onIntersectionChange(e[0]), { threshold: 0 });
       observer.observe(this._terminal.element);
       this.register({ dispose: () => observer.disconnect() });
     }
+
+    WebglRenderer._contextHelper.onRendererCreated(this);
   }
 
   public dispose(): void {
     this._renderLayers.forEach(l => l.dispose());
     this._terminal.screenElement.removeChild(this._canvas);
     super.dispose();
+    this._canvas.width = 0;
+    this._canvas.height = 0;
+
+    if (!this._gl.isContextLost()) {
+      console.log('Force context loss');
+      this.loseContext();
+    }
+    WebglRenderer._contextHelper.onRendererDestroyed(this);
+  }
+
+  public loseContext(): void {
+    this._gl.getExtension('WEBGL_lose_context').loseContext()
```

Comment:
Not supported in Chrome? https://developer.mozilla.org/en-US/docs/Web/API/WEBGL_lose_context/loseContext

Reply:
Nah, that seems outdated. It's definitely supported on Chrome and Chromium: https://codesandbox.io/embed/xpjjjmn1oq?view=preview